pkgs/applications/graphics/write_stylus/default.nix

```diff
@@ -0,0 +1,53 @@
+{ stdenv, lib, qt5, makeWrapper, fetchurl, makeDesktopItem }:
+stdenv.mkDerivation rec {
+  name = "write_stylus";
+
+  desktopItem = makeDesktopItem {
+    name = "Write";
+    exec = "Write";
+    comment = "a word processor for hadwriting";
+    desktopName = "Write";
+    genericName = "Write";
+    categories = "Office;Graphics";
+  };
+
+  src = fetchurl {
+    url = "http://www.styluslabs.com/download/write-tgz";
```

Comment:
Correct me if I am wrong, but the file pointed to by this url will probably change on the next update of Write, and will break the hash. If so, please replace it with an immutable url.